src/classes/ExpectInteger.cls

```diff
@@ -1,28 +1,57 @@
 public class ExpectInteger extends Expect {
     protected final Integer expected;
+    protected final Boolean negate;
+    protected final ExpectInteger parent;
 
     public ExpectInteger(Integer expected) {
         this.expected = expected;
+        this.negate = false;
+        this.parent = this;
+    }
+    public ExpectInteger(Integer expected, Boolean negate, ExpectInteger parent) {
+        this.expected = expected;
+        this.negate = negate;
+        this.parent = parent;
+    }
+    public ExpectInteger nnot {
+        get {
+            return negate ? this : new ExpectInteger(this.expected, true, this);
+        }
+    }
+    public ExpectInteger negated {
```

Comment:
Negated works for me. Any of them works for me really but negated has the additional benefit of not requiring renaming everything we've already put together. Appreciate your input! I'll get to work on it next week